### DISCUSSION TITLE
Cost and Pricing links after inserting new item are not to menu items

### DISCUSSION THREAD

**Discussion by dalers** [timschofield/webERP#258] [Problems / Bugs]
I created a new Item and am shown links to enter a) Cost and b) Pricing data, which I had expected would also be available as menu items, however I went through all the menu items and didn't find any that linked to these scripts. Do I understand this correctly? If so, why do these scripts _not_ need to be menu items?

My problem is that as a user with security role "Manufac/Stock Admin", I can create items but need to tell someone with the Accountant security role that cost and price data now needs to be entered (or curiously, someone who has the "Supplier (login only)" security role). If I logout to let the Accountant log in (we only have one computer in our company ;-) ), the page with the links is gone and the scripts aren't in the Menu system.

Am I not understanding something correctly?

Cheers,
Dale

![image](https://github.com/user-attachments/assets/5e42b260-936e-48b5-95c5-6bd65679736d)

![image](https://github.com/user-attachments/assets/f5a060c4-6566-4185-ba60-50f80d2b25dd)

**Comment by timschofield** [comment]
Hi Dale,

Those options should be available in the select product screen. Inventory
-> Select an Item

Thanks
Tim

On Wed, 27 Nov 2024 at 19:50, Dale Scott ***@***.***> wrote:

> I created a new Item and am shown links to enter a) Cost and b) Pricing
> data, which had expected the links to be to items in the menu system, but I
> don't think they are. Do I understand correctly, if so why do the pages not
> deserve to be menu items?
>
> My problem is that I'm a user with security role "Manufac/Stock Admin",
> and I now need to tell someone with the Accountant security role (or
> curiously someone with the "Supplier (login only)" security role), but if I
> log out to let the Accountant log in, the page with the links is gone and
> the pages aren't in the Menu.
>
> Am I misunderstanding what is happening?
>
> image.png (view on web)
> <https://github.com/user-attachments/assets/f5a060c4-6566-4185-ba60-50f80d2b25dd>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timschofield/webERP/discussions/258>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAL6LGXLP6L7PKDUHLRRYD2CYPCFAVCNFSM6AAAAABSTTK5WWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGU3DONJVHE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
www.weberp.org <http://www.weberpafrica.com>
@TimSchofield2
Blog: https://kwamoja.home.blog/

**Comment by dalers** [comment]
Right you are Tim, thanks.

![image](https://github.com/user-attachments/assets/a3028abc-dc8c-4b7b-bfdd-e0abf34a2ed6)
